ROADMAP.md

```diff
@@ -0,0 +1,10 @@
+# Roadmap for PySynphot
+
+Some time in 2020, make a last release and archive this package.
```

Comment:
@vglaidler , can the PR be accepted with this change then?
```suggestion
Some time in 2022, make a last release and archive this package.
```

Reply:
Yes, looks good to me. Thanks.